app-text/dblatex/dblatex-0.3.7.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="Transform DocBook using TeX macros"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+HOMEPAGE="http://dblatex.sourceforge.net/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~x86 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~x86-freebsd ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x86-macos ~x86-solaris"
```

Comment:
I'll change that.  I mean, I did that originally (well, ~amd64 and ~x86), but then repoman spat out fatal errors like

      dependency.bad [fatal]        44
       app-text/asciidoc/asciidoc-8.6.9-r2.ebuild: RDEPEND: ~alpha(default/linux/alpha/13.0)
    ['app-text/dblatex']

so I copied asciidoc's keywords.  I decided to check the devmanual and first it says "An ebuild must not depend upon any package that is of a lower keyword level than itself." (which I guess is where the repoman error comes from), but then further down under "Keywording New Packages" it says "Just file bugs for these architectures when a package is going to be a dependency of a package already keyworded.".  I take it this this what you are alluding to?